.github/workflows/main.yml

```diff
@@ -0,0 +1,23 @@
+name: Build and Deploy Storybook Documentation
+on: [push]
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1 # If you're using actions/checkout@v2 you must set persist-credentials to false in most cases for the deployment to work correctly.
+        with:
+          persist-credentials: false
+
+      - name: Install and Build 🔧
+        run: |
+          npm install
+          npm run build-storybook
+
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@3.6.1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BRANCH: gh-pages # The branch the action should deploy to.
```

Comment:
Is this the right branch?

Reply:
This will take the files out of the docs directory and move then to a branch called `gh-pages`.  I've configured the repo to use that branch for the github page that is used for documentation.